test/e2e/servicebindingrequest_test.go

```diff
@@ -506,23 +539,20 @@ func serviceBindingRequestTest(
 			CreateApp(todoCtx, t, f, cleanupOpts, deploymentNamespacedName, matchLabels)
 		case SBRStep:
 			// creating service-binding-request, which will trigger actions in the controller
-			sbr = CreateSBR(todoCtx, t, f, noCleanupOpts, sbrNamespacedName, resourceRef, matchLabels)
+			sbr = CreateSBR(todoCtx, t, f, noCleanupOpts, sbrNamespacedName, resourceRef, matchLabels, nil)
 		case SBREtcdStep:
-			sbr = CreateServiceBindingRequest(
-				todoCtx,
-				t,
-				ctx,
-				f,
-				serviceBindingRequestNamespacedName,
+			assertKeys = etcdSecretAssertion
+			sbr = CreateSBR(todoCtx, t, f,
+				noCleanupOpts,
+				sbrNamespacedName,
 				resourceRef,
 				matchLabels,
-				&v1.GroupVersionKind{
-					Group:   "etcd.database.coreos.com",
-					Version: "v1beta2",
-					Kind:    "EtcdCluster",
-				},
-				true,
-			)
+				func(sbr *v1alpha1.ServiceBindingRequest) {
+				setSBRBackendGVK(sbr, resourceRef, v1beta2.SchemeGroupVersion.WithKind(v1beta2.EtcdClusterResourceKind))
+				setSBRBindUnannotated(sbr, true)
+			})
```

Comment:
Indentation here looks funny. Did you `gofmt` it?

Reply:
did `gofmt`, indentation is this way because this is an function definition in the paramter